src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -0,0 +1,84 @@
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+
+import { secondaryPartnerList } from './secondaryPartnerList';
+
+const partnerGridStyle = {
+  xs: 12,
+  md: 6,
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexDirection: 'column',
+  pt: { xs: '40px', md: 0 }
+};
+
+const displaySecondaryPartners = ({
+  index,
+  company,
+  testimonial,
+  testimonialTwo,
+  testimonialAuthorTwo,
+  testimonialAuthor,
+  partnerLogo,
+  website
+}) => {
+  const gradientStyle =
+    index % 2 === 0
+      ? 'linear-gradient(90deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
+      : 'linear-gradient(270deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)';
```

Comment:
Personal opinion: I'm not sure about keeping the gradient style for mobile views. It's more consistent but for whatever reason feels unnecessary for small viewports.

Also, the OpenCommons logo isn't transparent. On desktop it doesn't show but on smaller screens it shows. Again, not a huge deal but it does visually clash. Especially when it's the others are transparent.

Reply:
This will be changed in a separate push after minor fixes. The OpenCommons logo is what I was given and as far as I'm aware there is no transparent background one (thats why i had it on the white side of gradient). The gradients I was on the fence for as well, after Scott's comment I'm going to do a mobile theme that keeps the logo on top and removes mobile gradient.